lib/transform/propertySetup.js

```diff
@@ -37,13 +37,13 @@ function propertySetup(property, name, path) {
   // Only do this once
   if (!property.original) {
     // Initial property setup
-    // Keep the original object properties so we can key off them in the transforms
+    // Keep the original object properties by which we can key off them in the transforms
```

Comment:
> // Keep the original object properties by which we can key off them in the transforms

// Keep the original object properties; we will key off of them in the transforms